### PR TITLE
Implement feature 'Scroll to top' after each redirecting

### PR DESCRIPTION
**Pull request by ivan-bonk**
- add function Trim breed for pet names in comparison page
    - add label hovering on a short pet name, shows full

### CODE REVIEW

**Review by Pashkaua — APPROVED** [review]

**Review by okhabr — APPROVED** [review]
👍

File: src/shared/components/loading-spinner/loading-spinner.module.scss [hunk 1]
```diff
@@ -18,16 +18,16 @@
  }
  .loading div {
    position: absolute;
-   width: 36px;
-   height: 36px;
+   width: 30px;
+   height: 30px;
    border-radius: 50%;
-   transform: translate(82px,82px) scale(1);
+   transform: translate(60px,60px) scale(1);
```
Comment (enheit) [inline src/shared/components/loading-spinner/loading-spinner.module.scss]: Is `scale(1)` has effect? I think it has no... Correct me if I wrong

File: src/shared/components/loading-spinner/loading-spinner.module.scss [hunk 2]
```diff
@@ -18,16 +18,16 @@
...
    background: #6f6f6f;
    animation: loading 1.333333333333333s infinite cubic-bezier(0,0.5,0.5,1);
  }
  .loading div:nth-child(1) {
    background: #5c4937;
-   transform: translate(150px,82px) scale(1);
+   transform: translate(111px,60px) scale(1);
```
Comment (enheit) [inline src/shared/components/loading-spinner/loading-spinner.module.scss]: Is `scale(1)` has effect? I think it has no... Correct me if I wrong

File: src/shared/components/scroll-to-top/scroll-to-top.component.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+export const ScrollToTop: React.FC = () => {
+    const pathname = useLocation();
+
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, [pathname]);
+    
+    return null;
+}
```
Comment (enheit) [inline src/shared/components/scroll-to-top/scroll-to-top.component.tsx]: nit: Add empty line to the end of the file

**Review by enheit — APPROVED** [review]

File: src/shared/components/scroll-to-top/scroll-to-top.component.tsx (hunk 3 above)
Comment (ivan-bonk) [reply src/shared/components/scroll-to-top/scroll-to-top.component.tsx]: Good point, i will fix it

File: src/shared/components/loading-spinner/loading-spinner.module.scss (hunk 1 above)
Comment (ivan-bonk) [reply src/shared/components/loading-spinner/loading-spinner.module.scss]: You right
In this case scale has no effect
But others 'scales' in this code are important

File: src/shared/components/loading-spinner/loading-spinner.module.scss (hunk 2 above)
Comment (ivan-bonk) [reply src/shared/components/loading-spinner/loading-spinner.module.scss]: You right
In this case scale has no effect

**Pull request merged**